src/screens/landingPage.js

```diff
@@ -1,15 +1,55 @@
 import React, { Component } from "react";
 import map from "../assets/images/map.PNG";
+import { ListGroup } from "react-bootstrap";
+import CategoryButtonGroup from "../components/CategoryButtonGroup/CategoryButtonGroup";
+
+const API = 'https://hn.algolia.com/api/v1/search?query=redux';
 
 class LandingPage extends Component {
+
+   constructor(props) {
+      super(props);
+      this.state = {
+         hits: []
+      };
+   }
+
+   componentDidMount() {
+      fetch(API)
+         .then(response => response.json())
+         .then(
+            data => this.setState({ hits: data.hits }));
+   }
+
+
    render() {
+      const { hits } = this.state;
       return (
          <div style={{
-            justifyContent: 'center', alignItems: 'center', margin: '0 auto',
-            width: '100vh', height: '100vh',
+            textAlign: 'center', margin: '0 auto',
+            width: '100%', height: '100%',
```

Comment:
Let's clean this up to either be in a const at the bottom of the file or at the very least a single line